main.tf

```diff
@@ -19,7 +19,7 @@ locals {
 }
 
 resource "google_compute_firewall" "allow-external-ssh" {
-  name    = "${local.prefix}openvpn-allow-external-ssh"
+  name    = "allow-external-ssh-${var.network}"
```

Comment:
network is not necesaary a name. It can be a self link. Reference the documentation. Please make that `openvpn-${var.name}-allow-external-ssh`